app/views/devise/registrations/edit.html.erb

```diff
@@ -0,0 +1,53 @@
+<h2>Edit <%= resource_name.to_s.humanize %></h2>
+
+<%= form_for(resource, as: resource_name, url: registration_path(resource_name), html: { method: :put }) do |f| %>
+  <%= render "devise/shared/error_messages", resource: resource %>
+
+  <div class="field">
+    <%= f.label :email %><br />
+    <%= f.email_field :email, autofocus: true, autocomplete: "email" %>
+  </div>
+
+  <div class="field">
+    <%= f.label :name %><br />
+    <%= f.text_field :name, autofocus: true, autocomplete: "name" %>
+  </div>
+
+  <div class="field">
+    <%= f.label :user_category %><br />
+    <%= f.select :user_category, User::USER_CATEGORY, autofocus: true, autocomplete: "user_category" %>
```

Comment:
Instead of getting these from model. We should use helper method `User::USER_CATEGORY`
Also don't use autocomplete for dropdown or select field type, instead use the value which is saved in the DB as this is the edit form.
If its working fine then leave it

Reply:
I am using the helper method instead of User::USER_CATEGORY.